October_2020/DS&Algo_Series

```diff
@@ -10,7 +10,7 @@ All the instructors are already placed in FAANG companies and if you're willing
 
 https://www.youtube.com/playlist?list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ
 
-Thanku,
+Thank You,
 Regards,
```

Comment:
Can you please revert it?

Reply:
Yeah Sure! I have changed it. Kindly check.